Demo/Form1.cs

```diff
@@ -30,6 +32,10 @@ public Form1()
 			incrementalSearcher1.FindReplace = MyFindReplace;
 
 			findAllResultsPanel1.Scintilla = scintilla1;
+			
+			// The default line number is 1, anything else is a custom line number
+			customStartingLineNumber = 8;
```

Comment:
The demo app should have an option to enable the custom line numbering. Either one way &rarr; button click or two-way such as:
![image](https://github.com/VPKSoft/ScintillaNET-FindReplaceDialog/assets/40712699/bdc5ba81-0559-48f2-9cce-10277babd46b)

Reply:
I've added a new commit to enable\disable the custom line numbering, but I'm new to git and GitHub so I'm not sure if I've added correctly.

Reply:
This looks good.